pages/news/2020-05-15-you-dont-need-to-pay-for-mixxx.html

```diff
@@ -0,0 +1,77 @@
+title: You don't need to pay for Mixxx!
+author: Holzhaus
+date: 2020-05-15 13:00:00
+
+{% extends "post.html" %}
+
+{% block post %}
+
+{% load markup %}
+{% filter markdown %}
+
+It recently came to our attention that some sellers distribute boxed versions of Mixxx on Ebay and Amazon.
+While it's not illegal to do this per se, we are not affiliated with these sellers and neither earn money from this nor endorse this in any way.
+Instead, you can download the latest version of Mixxx from the [download page on our website]({% url "/download.html" %}) for free.
+
+If you want to support Mixxx financially, consider donating - we'll appreciate it!
+You can use the *Donate* button on the download page.
+
+
+### Mixxx was, is and always will be free
+
+In contrast our commercial competitors like Serato or VirtualDJ that may offer "free" featured-reduced versions of their Software, the Mixxx developers do not impose any artificial restrictions on their software.
+There is no "Pro Edition" or Premium version of Mixxx, even if it says so in the description of the online shop.
+
+The Mixxx developers don't sell Mixxx, and if you "buy" Mixxx somewhere online you'll likely just get the free Mixxx version that the seller downloaded from our website and burned onto a CD.
+Even worse, some of these paid Mixxx versions might even contain ads or malware according to Amazon reviews.
+Just use the free download from our Website to get the ad-free, malware-free, most recent version of Mixxx.
+
+
+### Mixxx is free as in freedom, not just free as in "free beer"
+
+You don't need to pay money to use Mixxx, but there's more:
+It's source code is accessible to everyone and if you know how to code, you can modify it and customize Mixxx to a degree that is simply not possible with any closed-source software.
+Even if you did get your hands on the source code of Serato DJ Pro/Lite, Rekordbox, Traktor or VirtualDJ (which is not available their users), it would be illegal for you to modify the program or redistribute it.
+
+Mixxx, on the other hand, is [free software](https://www.gnu.org/philosophy/free-sw.html) and grants you the four essential freedoms:
+
+1. The freedom to run the program as you wish, for any purpose.
+2. The freedom to study how the program works, and change it so it does your computing as you wish. Access to the source code is a precondition for this.
+3. The freedom to redistribute copies so you can help others.
+4. The freedom to distribute copies of your modified versions to others. By doing this you can give the whole community a chance to benefit from your changes. Access to the source code is a precondition for this.
```

Comment:
0 through 3

Reply:
Yeah, because arrays start at 0... But will our non-technical users understand that?

Reply:
This is how it has always been numbered on https://www.gnu.org/philosophy/free-sw.html . If we're copying the text verbatim, let's copy the numbers as they are.

> The reason they are numbered 0, 1, 2 and 3 is historical. Around 1990 there were three freedoms, numbered 1, 2 and 3. Then we realized that the freedom to run the program needed to be mentioned explicitly. It was clearly more basic than the other three, so it properly should precede them. Rather than renumber the others, we made it freedom 0.

Reply:
The markdown parser changes this back to 1-4 anyway. I switch to an unnumbered list.

Reply:
No numbers are better than wrong numbers.